src/utils/axios.js

```diff
@@ -107,38 +103,49 @@ async function getFormsAndFields(basicAuth, api, baseUrl) {
                     basicAuth,
                     api,
                     baseUrl,
-                    currentForm
+                    currentForm,
+                    reporter
                 )
 
                 formObj['form-' + currentForm.id] = form
             }
         } else {
-            log(chalk.bgRed('We could not find any forms. Have you made any?'))
+            reporter.error('We could not find any forms. Have you made any?')
         }
 
         return formObj
     }
     return false
 }
 
-function apiErrorHandler(error) {
+function apiErrorHandler(error, reporter) {
     if (error.response) {
         // The request was made and the server responded with a status code
         // that falls out of the range of 2xx
-        log(chalk.bgRed('Request was made, but there was an issue'))
-        log(error.response.data)
-        log(error.response.status)
-        log(error.response.headers)
+        reporter.panicOnBuild(
+            'Request was made, but there was an issue',
+            new Error(`Error ${error.response.status} from GraityForms API`)
+        )
+
+        // log(error.response.data)
```

Comment:
What are your thoughts here? Are these not needed? Would be good to be able to pass through to provide more context to the issue.

Reply:
:man_shrugging: I don't remember my thought process at this point, We can add them back to the log if you think they'd be helpful.